[2025-02-23T14:00:03.257+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T14:00:03.288+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-23T13:00:00+00:00 [queued]>
[2025-02-23T14:00:03.299+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-23T13:00:00+00:00 [queued]>
[2025-02-23T14:00:03.300+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-23T14:00:03.318+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2025-02-23 13:00:00+00:00
[2025-02-23T14:00:03.325+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=19458) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-23T14:00:03.327+0000] {standard_task_runner.py:72} INFO - Started process 19462 to run task
[2025-02-23T14:00:03.328+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'london_weather_pipeline', 'transform_weather_data', 'scheduled__2025-02-23T13:00:00+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/london_weather_pipeline/london_weather_etl.py', '--cfg-path', '/tmp/tmpocmtccuj']
[2025-02-23T14:00:03.329+0000] {standard_task_runner.py:105} INFO - Job 120: Subtask transform_weather_data
[2025-02-23T14:00:03.400+0000] {task_command.py:467} INFO - Running <TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-23T13:00:00+00:00 [running]> on host ee2081ae3a04
[2025-02-23T14:00:03.552+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='london_weather_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T13:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-23T13:00:00+00:00'
[2025-02-23T14:00:03.553+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T14:00:03.557+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': '51.5074', 'longitude': '-0.1278', 'temperature': 10.9, 'windspeed': 21.3, 'winddirection': 202, 'weathercode': 3}
[2025-02-23T14:00:03.608+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T14:00:03.609+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=london_weather_pipeline, task_id=transform_weather_data, run_id=scheduled__2025-02-23T13:00:00+00:00, execution_date=20250223T130000, start_date=20250223T140003, end_date=20250223T140003
[2025-02-23T14:00:03.701+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T14:00:03.736+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T14:00:03.739+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
